store/types.go

```diff
@@ -24,23 +22,11 @@ type Store interface {
 	// GetBlockByHash returns block with given block header hash, or error if it's not found in Store.
 	GetBlockByHash(ctx context.Context, hash types.Hash) (*types.SignedHeader, *types.Data, error)
 
-	// SaveBlockResponses saves block responses (events, tx responses, validator set updates, etc) in Store.
-	SaveBlockResponses(ctx context.Context, height uint64, responses *abci.ResponseFinalizeBlock) error
```

Comment:
where will this be moved?

Reply:
It's being removed, as the go-execution api doesn't return the responses for the txs, meaning these will be handled on the execution layer

Reply:
that part makes sense, but i believe the "block" is still stored in rollkit?